backend/osmosewebsite/management/commands/seed.py

```diff
@@ -9,6 +9,8 @@
 from django.core.management.base import BaseCommand
 
 from backend.osmosewebsite.models import TeamMember, News, Collaborator, Project
+from backend.osmosewebsite.models import TeamMember, Trap, Collaborator, Project
```

Comment:
Why do you import twice the same classes? Trap should be added to the initial import line